src/content/en/updates/2017/02/improved-add-to-home-screen.md

```diff
@@ -141,42 +141,82 @@ Note: The `scope` in the Web App Manifest, is not the same as the `scope` of
 your Service Worker. It is possible for one installed Web Application to be
 controlled by several different Service Workers.
 
-## Navigating outside of my progressive web app
+## Navigating outside of your progressive web app
+
+It is important for users to feel like they are using an app-like experience
+when using your progressive web app. This means that we need to handle
+navigations sensetively. If the user navigates to another origin then your
+progressive web app should not act like a container, instead a new context will
+be opened for the user.
+
+The `scope` attribute also is used to define the reach of your web application
+within the origin. Any navigation to a url inside your origin, but outside the
+scope will cause a new web context to be opened for the user. This allows you to
+have multiple progressive web apps per origin.
+
+## Effectively managing your app's scope
+
+The `scope` attribute in your web app manifest is one of the most important
+things that you will need to do to ensure your app functions as expected.
+
+The `scope` controls the url structure that encompasses all the entry and exit
+points in your web app. Your `start_url` needs to reside in the your `scope`.
```

Comment:
This is different from service worker scope, isn't it? You should address this somewhere.